en/docs/get-started/open-packages.md

```diff
@@ -0,0 +1,15 @@
+# Opening Ballerina Packages
+
+You can work on a single [Ballerina Package](https://ballerina.io/learn/package-references/) at a time or use [Multi-Root workspaces]((https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina)) of VS Code to work on serverl related Ballerina Packages at the same time. 
+
+## Opening a Ballerina Package
+
```

Comment:
Use one of the following options to open a Ballerina package:

- From command line, execute `code <path-to-package-dir>`.
- From VS Code, click **File > Open Folder ...**.